docs/docs/audit-with-lighthouse.md

```diff
@@ -1,51 +1,51 @@
 ---
-title: Audit with Lighthouse
+title: Auditar con Lighthouse
 ---
 
-Quoting from the [Lighthouse website](https://developers.google.com/web/tools/lighthouse/):
+Citando el [sitio web de Lighthouse](https://developers.google.com/web/tools/lighthouse/):
 
-> Lighthouse is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps (PWAs), and more.
+> Lighthouse es una herramienta automatizada de código abierto diseñada para mejorar la calidad de tus apps web. Puedes ejecutarla en cualquier página web, pública o que requiera autenticación. Tiene auditorías de rendimiento, accesibilidad, aplicaciones web progresivas (_PWAs_), y más.
 
-Lighthouse is included in Chrome DevTools. Running its audit -- and then addressing the errors it finds and implementing the improvements it suggests -- is a great way to prepare your site to go live. It helps give you confidence that your site is as fast and accessible as possible.
+Lighthouse está incluida en las herramientas de desarrollo de Chrome. Ejecutar su auditoría -- y luego abordar los errores que encuentra e implementar las mejoras que sugiere -- es una excelente manera de preparar tu sitio para salir al aire. Te ayuda a confiar en que tu sitio es lo más rápido y accesible posible.
```

Comment:
Creo que una tradución a un castellano más neutro de `is a great way to prepare your site to go live.` sería `es una excelente manera de preparar tu sitio para ponerlo online`. 

En lugar de `ponerlo online` también se podría decir `salir a producción` o `ofrecerlo online`.